Migrate several devices to none/on effects (and breathing mono effect)

z3ntu opened the pull request:
Now that the major UIs support this effect, let's update the daemon to use those new methods.

---

~~^^ above commit message is for the future. To be merged when polychromatic and RazerGenie have new releases~~

**On** effect

* [x] https://github.com/polychromatic/polychromatic/pull/482 (https://github.com/polychromatic/polychromatic/commit/dcbd3a82cc567c893ae03d3a5dc153ffc001c15d)
* [x] https://github.com/z3ntu/libopenrazer/commit/1f25aab0f2df81acdd36e0a405229463d31c2c6d

**Breath mono-color** effect

* [x] https://github.com/polychromatic/polychromatic/pull/486
* [x] https://github.com/z3ntu/RazerGenie/commit/03e241b1224b6ec13bae97014a0882298fcf1b4f
* [x] https://github.com/z3ntu/libopenrazer/commit/143d51b38d3612a6775c65fcf1e56815d72c6aa9

CC @lah7

z3ntu committed:
Move Razer Naga 2012 & Razer Naga 2014 to none/on effects

Now that the major UIs support this effect, let's update the daemon to
use those new methods. And then we can also remove the unused bits from
the driver.

lah7 reviewed (approved):
Works here under the fake driver 👍🏼 

We just need to fix the tests to recognise this new effect.

![Screenshot_20230619_214634](https://github.com/openrazer/openrazer/assets/13032135/ff008402-ed5f-4a9a-aa11-387c9a861f52)

z3ntu committed:
Move Razer Abyssus to none/on effects

z3ntu committed:
Move Razer Imperator to none/on effects

z3ntu committed:
Move Razer Naga Hex & Razer Naga Hex Red & Razer Taipan to none/on effects

z3ntu committed:
Move Razer DeathAdder 3.5G to none/on effects

z3ntu committed:
driver: Simplify set_led_state for DeathAdder 3.5G

z3ntu committed:
Move Razer Orochi 2011 to none/on effects

z3ntu committed:
Move Razer Abyssus 1800 and Razer DeathAdder 1800 to none/on effects

z3ntu committed:
Move Razer Abyssus 2000 to none/on effects

z3ntu committed:
Move Razer Ouroboros to none/on effects

z3ntu committed:
Move Razer Orochi 2013 to none/on effects

z3ntu committed:
Move Razer Orochi Chroma to none/on effects

A bit of a weird mix between matrix and led_state somehow...

z3ntu committed:
Move Razer DeathAdder 2000 to none/on and new breath mono effect

z3ntu committed:
pylib: add support for breath mono effect

As added in the last commit, add the effect to pylib also.

z3ntu committed:
daemon: Clean up some now-unused dbus methods

z3ntu committed:
daemon: Add todo note into bw2013 dbus methods

z3ntu committed:
razermouse: move some request package assignment around

Move it where it makes more sense, and add a default case to catch
faulty uses.
